src/views/PillarAnalysis/AnalyticalStatementInput/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useCallback } from 'react';
+import { GrDrag } from 'react-icons/gr';
+import { BiBarChartSquare } from 'react-icons/bi';
+import {
+    IoClose,
+    IoAdd,
+    IoCheckmarkCircleSharp,
+} from 'react-icons/io5';
+import { _cs, randomString } from '@togglecorp/fujs';
+import {
+    QuickActionButton,
+    TextArea,
+} from '@the-deep/deep-ui';
+import {
+    PartialForm,
+    useFormArray,
+    useFormObject,
+    Error,
+} from '@togglecorp/toggle-form';
+
+import { AnalyticalStatementType, AnalyticalEntryType } from '../schema';
+import AnalyticalEntryInput from './AnalyticalEntryInput';
+
+import styles from './styles.scss';
+
+interface AnalyticalStatementInputProps {
+    className?: string;
+   value: PartialForm<AnalyticalStatementType>;
+   error: Error<AnalyticalStatementType> | undefined;
+   onChange: (value: PartialForm<AnalyticalStatementType>, index: number) => void;
+   onRemove: (index: number) => void;
+   index: number;
+}
+
+function AnalyticalStatementInput(props: AnalyticalStatementInputProps) {
+    const {
+        className,
+        value,
+        error,
+        onChange,
+        onRemove,
+        index,
+    } = props;
+
+    const onFieldChange = useFormObject(index, value, onChange);
+
+    const {
+        // onValueChange: onAnalyticalEntryChange,
+        onValueRemove: onAnalyticalEntryRemove,
+    } = useFormArray('analyticalEntries', value.analyticalEntries ?? [], onFieldChange);
+
+    const handleAnalyticalEntryAdd = useCallback(
+        () => {
+            const uuid = randomString();
+            const newAnalyticalEntry: PartialForm<AnalyticalEntryType> = {
+                uuid,
+                // FIXME: add order
+                order: 0,
+                entry: Math.ceil(Math.random() * 100),
```

Comment:
why are we using this random number for? Isn't uuid sufficient?

Reply:
This is temporary. Will remove this after we can drag/drop the entry from sidepane